i18n4k-gradle-plugin/src/main/kotlin/de/comahe/i18n4k/gradle/plugin/GenerateI18n4kFilesTask.kt

```diff
@@ -5,17 +5,15 @@ import de.comahe.i18n4k.generator.GenerationTargetPlatform
 import de.comahe.i18n4k.generator.I18n4kProcessor
 import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.InputDirectory
-import org.gradle.api.tasks.OutputDirectory
-import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.*
 import java.io.File
 
 
 @Suppress("unused")
 open class GenerateI18n4kFilesTask : DefaultTask() {
 
 
+    @Nested
     lateinit var config: I18n4kExtension
```

Comment:
Hi @saruye,
Thanks for your contribution. But an `@Internal` annotation at the `config` member would be the better solution, as `config` is used for the other parameters of the task.